lib/sgf/gtp_writer.rb

```diff
@@ -0,0 +1,31 @@
+class SGF::GtpWriter < SGF::Writer
+
+  private
+
+  def gtp_move(node)
+    pps = node.properties
+    return "boardsize #{pps['SZ']}\nclear_board" if pps['SZ']
+    return "" if pps.size > 1 || (pps.keys != ["B"] && pps.keys != ["W"])
+
+    if pps.values == [""]
+      gtp_pos = "pass"
+    else
+      pos = pps.values.first.bytes
```

Comment:
Do we really have to get the bytes here or could we just work with the actual string value?

Reply:
i need the bytes to make the calculation below